### PR TITLE
EZP-27508: Remove Translation from Version

### PR DESCRIPTION
**Pull request by DominikaK**
https://jira.ez.no/browse/EZP-27508

(from 1.12)

Based on https://github.com/ezsystems/ezpublish-kernel/blob/1ffd08721cbd267a7d1c7cda20cac03200a092a3/doc/specifications/translations/removal.md#removing-the-specific-language-translation-from-a-content-object-version-draft

### CODE REVIEW

**Comment by DominikaK** [comment]
ping @alongosz

File: docs/api/public_php_api.md [hunk 1]
```diff
@@ -1075,6 +1075,48 @@ $contentUpdateStruct->setField( 'body', $newfrenchbody );
...
+
+```
+public function deleteTranslationFromDraft(VersionInfo $versionInfo, $languageCode);
+```
+
+This method returns a Content draft without the specified translation.
+
+!!! note
+
+    To remove main the translation, the main language needs to be changed manually
```
Comment (alongosz) [inline docs/api/public_php_api.md]: Typo: `the main`, right?

File: docs/api/public_php_api.md [hunk 2]
```diff
@@ -1075,6 +1075,48 @@ $contentUpdateStruct->setField( 'body', $newfrenchbody );
...
+
+
+The PHP API consumer is responsible for creating a Content item version draft and publishing it after translation removal.
+
+Since the returned Content draft is to be published, both search and HTTP cache are already handled
+by `PublishVersion` slots once the call to `publishVersion()` is made.
+
+Example:
+
+```
```
Comment (alongosz) [inline docs/api/public_php_api.md]: adding `php` keyword at the end of this line should colour the syntax. Not sure if this works for ReadTheDocs, though (depends on markdown renderer)

File: docs/api/public_php_api.md [hunk 3]
```diff
@@ -1075,6 +1075,48 @@ $contentUpdateStruct->setField( 'body', $newfrenchbody );
...
 Since we don't specify a locale for the last two fields, they are set for the `UpdateStruct`'s `initialLanguageCode`, fre-FR.
 
+#### delete translations
+
+##### delete translations from a Content item version
+
+To delete translations from a Content item version, use the `deleteTranslationFromDraft` method on `ContentService`.
+
+```
+public function deleteTranslationFromDraft(VersionInfo $versionInfo, $languageCode);
```
Comment (alongosz) [inline docs/api/public_php_api.md]: Just got an idea, let's use PHP7+ syntax to document method:
`public function deleteTranslationFromDraft(VersionInfo $versionInfo, string $languageCode) : Content`

File: docs/api/public_php_api.md (hunk 2 above)
Comment (DominikaK) [reply docs/api/public_php_api.md]: Even better, this now ensures ReadTheDocs will recognize the language and colour appropriately.

**Comment by DominikaK** [comment]
@alongosz, fixed according to your comments.

**Comment by alongosz** [comment]
Thanks @DominikaK :tada:

**Pull request merged**